CHANGELOG.md

```diff
@@ -4,6 +4,14 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/). This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-[x.x.x]
+## [1.1.0] - 2022-04-27
+###Added
+- Support for Service Mesh Service
+- Supports OLM version 0.20.0
+- Supports operator bundle upgrade
 
-- Initial Release of OSOK
\ No newline at end of file
+## [1.0.0] - 2021-08-31 Initial Release
+###Added
+- Support for Autonomous Database Service
+- Support for Oracle Streaming Service
+- Support for MySQL DB System Service
```

Comment:
Can you add the previous OLM version here.

Reply:
Done